corehq/toggles.py

```diff
@@ -67,12 +67,20 @@ def __init__(self, slug, label, tag, namespaces=None, help_link=None,
         self.always_disabled = always_disabled or set()
         self.enabled_for_new_domains_after = enabled_for_new_domains_after
         self.enabled_for_new_users_after = enabled_for_new_users_after
+        # pass in a set of environments where this toggle applies
+        self.relevant_environments = relevant_environments
```

Comment:
you could initialize this as `relevant_environments or []` and get rid of the null check later

Reply:
Actually I need the null check - if it's not specified (or empty), then we need to treat it as if all environments are listed.